homeassistant/components/min_max/config_flow.py

```diff
@@ -72,3 +75,45 @@ class ConfigFlowHandler(SchemaConfigFlowHandler, domain=DOMAIN):
     def async_config_entry_title(self, options: Mapping[str, Any]) -> str:
         """Return config entry title."""
         return cast(str, options["name"]) if "name" in options else ""
+
+    @callback
+    @staticmethod
+    def async_setup_preview() -> None:
+        """Set up preview WS API."""
+        hass = async_get_hass()
+        websocket_api.async_register_command(hass, ws_preview_min_max)
+
+
+@callback
+@websocket_api.websocket_command(
+    {
+        vol.Required("type"): "min_max/preview",
+        vol.Required("flow_id"): str,
+        vol.Required("flow_type"): str,
+        vol.Required("user_input"): dict,
+    }
+)
+def ws_preview_min_max(
+    hass: HomeAssistant, connection: websocket_api.ActiveConnection, msg: dict[str, Any]
+) -> None:
+    """Generate a preview."""
+    if msg["flow_type"] == "config_flow":
+        validated = CONFIG_SCHEMA(msg["user_input"])
+        name = validated["name"]
+    else:
+        validated = OPTIONS_SCHEMA(msg["user_input"])
+        flow_status = hass.config_entries.options.async_get(msg["flow_id"])
+        config_entry = hass.config_entries.async_get_entry(flow_status["handler"])
+        assert config_entry
```

Comment:
Don't assert. We should send a not found error.

Reply:
Fixed